SPEC.md

```diff
@@ -0,0 +1,111 @@
+# adminbot spec
+
+This file documents both what adminbot does ambiently and what commands it responds to.
+
+## Behaviors
+
+### Welcome message
+
+adminbot knows what users it has seen and which ones it hasn't. When an unrecognized user joins
+`#tildetown`, adminbot welcomes them and gives some helpful hints.
+
+```
+--> | wren has joined
+<adminbot> Welcome, wren! I hope you're well. If you haven't seen it yet, please read over our wiki
```

Comment:
When we were talking about `probot`, it was mentioned that, if you `/notice wren [#tildetown] Welcome, wren! ...`, it'll show up as
```
PvNotice(adminbot): Welcome, wren! ...
```
for them only (weechat pops it in the same buffer as #tildetown, which is handy). This could be helpful if you don't want other users to see the welcome messages, although I guess the welcome messages could be helpful to help point out new users... (or we could have both?!)

Reply:
i was thinking these would definitely be public so users knew that someone coming in was new.

Reply:
ahh, okay - fair enough then